### DISCUSSION TITLE
引入Waline评论区出现接口调用错误的问题

### DISCUSSION THREAD

**Discussion by nonhana** [theme-shoka-x/hexo-theme-shokaX#253] [Q&A]
如图所示，我的Waline在Zeabur上部署后，在_config.shokaX.yml文件的指定地方引入：
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/105428118/01b9a4db-2bc6-44e4-aacc-20a37e5caeba)
然后hexo s，访问项目，出现接口请求路径错误：
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/105428118/0a127772-569d-43c0-9562-30c064285ab4)
我打开这个评论区本身的访问url（即根路径/），请求是成功的：
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/105428118/96c8e990-84f9-4385-81a0-750e5346410d)
可见就是多了一个/api路径。
由于是在shokaX主题本身的配置文件中进行引入的，我考虑可能是这个主题本身的问题，因此在此处进行提问。
或者有能够更改接口请求路径的配置选项，也可以提供一下~

**Comment by FidStyle** [comment]
与我配置相同，建议检查shokax版本以及waline部署的版本是否为最新版。
Zeabur部署的版本不一定为waline最新版。
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/1e73ecdb-a3b0-457e-ae22-ef806a4f9407)
![image](https://github.com/theme-shoka-x/hexo-theme-shokaX/assets/115857087/028f9f56-5575-4d02-a5dd-0824917a96c2)
